_posts/2022-08-25-day-in-the-life-of-a-release-manager.md

```diff
@@ -0,0 +1,55 @@
+
+It all starts with a gif. My manager asks in our team Slack if anyone would like to volunteer to be the release manager of our plugins. My mind immediately goes to the scene in the The Hunger Games where Katniss volunteers as tribute. I want to send the gif, but that means adding more to my plate. In addition to it being my first release, this one has big features, so I ask my colleagues if it will be manageable. They tell me that the work of a release manager starts after code freeze and that there will be no overlap between my responsibilities as an individual contributor and a release manager. The responsibilites would just be to complete a list of tasks for each of our plugins and report to the overall release manager. I am quickly convinced, and I hit send on the gif.
+
+A couple weeks go by with no updates until a couple days before feature freeze, I am given my first mission: Add all of the plugins to the 1.3 release manifest with 1.3 branches. My actual first mission: Figure out what a release manifest is.
+
+What is a release manifest? I turn to the [results](https://dzone.com/articles/release-snapshots-smart#:~:text=A%20release%20manifest%20contains%20the,signatures%20from%20sender%20and%20receiver.) of a quick Google search.
+
+*A release manifest contains the collection of versioned stuff that is being deployed, configuration settings, and approvals. What, how, where, and who approved it. This is similar to a shipping form listing out the boxes sent, value and contents of the goods, destination, and signatures from sender and receiver.*
+
+I find the release manifest for OpenSearch within a repository called [opensearch-build](https://github.com/opensearch-project/opensearch-build) within a folder conveniently called [manifests](https://github.com/opensearch-project/opensearch-build/tree/main/manifests). I start reading all of the docs I can find related to this release and previous ones. 
+
+After forking the [repository](https://github.com/opensearch-project/opensearch-build), I check out a branch, add my team’s plugins to the manifest, and create a [pull request](https://github.com/opensearch-project/opensearch-build/issues/889#issuecomment-1036510660) (PR).
+
+Two weeks before release, [GitHub issues](https://github.com/opensearch-project/observability/issues/503) are created in each repository with a checklist for each (the actual checklist that was created for this version release is pictured below). I self-assign the issue and check off some of the completed tasks. In hindsight, it seems obvious that I would know that checking off a task does not mean that it is actually completed. But this only becomes clear to me when I wake up one morning to the continuous integration (CI) builds failing.
+
+![Image: Figure 1]({{ site.baseurl }}/assets/media/blog-images/2022-08-24-day-in-the-life-of-a-release-manager/release-checklist-1.png){: .img-fluid }
```

Comment:
"2022-08-24" - looks like this should be 2022-08-25 on each of your images

Reply:
Good catch, thank you!